package.json

```diff
@@ -13,18 +13,19 @@
   },
   "repository": {
     "type": "git",
-    "url": "git://github.com/mappum/nomic.git"
+    "url": "git://github.com/botwillacceptanything/botwillacceptanything.git"
   },
   "keywords": [
     "nomic",
     "autonomous",
     "sentient",
-    "skynet"
+    "skynet",
+    "botwillacceptanything"
   ],
-  "author": "Matt Bell <mappum@gmail.com>",
+  "author": "Bot W. Acceptantyhing <botwillacceptanything@gmail.com>",
```

Comment:
Hmm I wish I could edit the PR without wiping out the current votes.  I will make that my next PR!